origen_testers.gemspec

```diff
@@ -34,4 +34,5 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'origen_stil', '>= 0.2.1'
   spec.add_runtime_dependency "ast", "~> 2"
   spec.add_runtime_dependency "sexpistol", "~> 0.0"
+  spec.add_runtime_dependency 'rubyzip', '1.3.0'
```

Comment:
Can you remove this, I've pushed an update to master which should hopefully fix it